Source/NSTreeController.m

```diff
@@ -22,55 +22,92 @@
 
    You should have received a copy of the GNU Lesser General Public
    License along with this library; see the file COPYING.LIB.
-   If not, see <http://www.gnu.org/licenses/> or write to the 
-   Free Software Foundation, 51 Franklin Street, Fifth Floor, 
+   If not, see <http://www.gnu.org/licenses/> or write to the
+   Free Software Foundation, 51 Franklin Street, Fifth Floor,
    Boston, MA 02110-1301, USA.
-*/ 
-/* 
-   NSTreeController.h
+*/
 
-   The tree controller class.
+#import <Foundation/NSArchiver.h>
+#import <Foundation/NSArray.h>
+#import <Foundation/NSDictionary.h>
+#import <Foundation/NSIndexPath.h>
+#import <Foundation/NSKeyedArchiver.h>
+#import <Foundation/NSKeyValueCoding.h>
+#import <Foundation/NSKeyValueObserving.h>
+#import <Foundation/NSNotification.h>
+#import <Foundation/NSString.h>
+#import <Foundation/NSSortDescriptor.h>
 
-   Copyright (C) 2012 Free Software Foundation, Inc.
+#import "AppKit/NSOutlineView.h"
+#import "AppKit/NSKeyValueBinding.h"
+#import "AppKit/NSTreeController.h"
+#import "AppKit/NSTreeNode.h"
 
-   Author:  Gregory Casamento <greg.casamento@gmail.com>
-   Date: 2012
-   
-   This file is part of the GNUstep GUI Library.
+#import "GSBindingHelpers.h"
+#import "GSFastEnumeration.h"
+#import "GSControllerTreeProxy.h"
 
-   This library is free software; you can redistribute it and/or
-   modify it under the terms of the GNU Lesser General Public
-   License as published by the Free Software Foundation; either
-   version 2 of the License, or (at your option) any later version.
+@interface NSOutlineView (__NSTreeController_private__)
 
-   This library is distributed in the hope that it will be useful,
-   but WITHOUT ANY WARRANTY; without even the implied warranty of
-   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.	 See the GNU
-   Lesser General Public License for more details.
+- (NSArray *) _selectedIndexPaths;
 
-   You should have received a copy of the GNU Lesser General Public
-   License along with this library; see the file COPYING.LIB.
-   If not, see <http://www.gnu.org/licenses/> or write to the 
-   Free Software Foundation, 51 Franklin Street, Fifth Floor, 
-   Boston, MA 02110-1301, USA.
-*/ 
+@end
 
-#import <Foundation/NSArray.h>
-#import <Foundation/NSIndexPath.h>
-#import <Foundation/NSString.h>
-#import <Foundation/NSSortDescriptor.h>
+@implementation NSOutlineView (__NSTreeController_private__)
+
+- (NSArray *) _selectedIndexPaths
```

Comment:
I don't see where this method ever gets called. What do we need it for?

Reply:
Removed